ext/opcache/jit/zend_jit.h

```diff
@@ -117,6 +117,9 @@ typedef struct _zend_jit_globals {
 	uint8_t bad_root_cache_stop[ZEND_JIT_TRACE_BAD_ROOT_SLOTS];
 	uint32_t bad_root_slot;
 
+	uint32_t function_compilation_successes;
+	uint32_t function_compilation_failures;
+
```

Comment:
jit_globals structure is not shared across processes, so each process is going to keep (and show) only it's own number of compiled and failed functions.